cubeviz/controls/tests/test_slice_controller.py

```diff
@@ -69,7 +69,7 @@ def test_wavelength_slider(cubeviz_layout, slice_index, cube_bounds):
     assert_all_viewer_indices(cubeviz_layout, slice_index)
 
     # Make sure that wavelength text matches slice value
-    wavelength_text = "{:.3}".format(cube_bounds['wavelengths'][slice_index])
+    wavelength_text = "{:.4e}".format(cube_bounds['wavelengths'][slice_index])
```

Comment:
Everything looks good. Tested and works.  But, if we are using @astrofrog's code for determining the proper number of sig-figs, why is this hard coded here?  (there is another place too, but I suspect it will be the same answer).